kolibri/plugins/html5_app_renderer/assets/src/views/index.vue

```diff
@@ -0,0 +1,112 @@
+<template>
```

Comment:
Woops, this file should be removed. It was renamed to Html5AppRendererIndex.vue

Reply:
Ugh, thought I checked for this, sorry!